specs-go/version.go

```diff
@@ -150,12 +150,20 @@ func requiresV110(spec *Spec) bool {
 		}
 	}
 
+	if len(spec.ContainerEdits.NetDevices) != 0 {
```

Comment:
This really is a big nit: At other places where we're checking non-empty, we explicilty check `> 0` and not `!= 0`. Not going to block on this though.